build/spec.c

```diff
@@ -451,6 +453,7 @@ const char * rpmSpecGetSection(rpmSpec spec, int section)
 	case RPMBUILD_BUILD:	return getStringBuf(spec->build);
 	case RPMBUILD_INSTALL:	return getStringBuf(spec->install);
 	case RPMBUILD_CHECK:	return getStringBuf(spec->check);
+	    //case RPMBUILD_POSTBUILD:	return getStringBuf(spec->postbuild);
```

Comment:
any reason this is commented out?

Reply:
left there as a reminder that RPMBUILD_POSTBUILD is still missing as a constant.